src/redux/cities/citiesSlice.js

```diff
@@ -0,0 +1,49 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import getWeatherData from '../api/api';
+
+export const fetchCities = createAsyncThunk(
+  'cities/fetchCities',
+  async (cityNames) => {
+    // console.log(cityNames);
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code bulky and unprofessional